File: PAT (Basic-Level)/C/1006.c
#include <stdio.h>
int main()
{
	int n, i, tem;
	scanf("%d", &n);
	if(n >= 100){
		tem = n / 100;
		n = n - tem*100;
		for( i = 0 ; i < tem ; i++)
		printf("B");
	}
	
	if(n >= 10){
		tem = n / 10;
		n = n - tem*10;
		for( i = 0 ; i < tem ; i++)
		printf("S");
	}
	
	if(n != 0){
		for( i = 1 ; i <= n ; i++)
		printf("%d", i);
	}
	
	return 0;
}

File: PAT (Basic-Level)/Python/1006.py
#!/usr/bin/python3
# -*- coding: utf-8 -*-

n = int(input())
if n >= 100:
    tem = int(n / 100)
    n = n - tem*100
    for i in range(tem):
        print("B", end = "")

if n >= 10:
    tem = int(n / 10)
    n = n - tem*10
    for i in range(tem):
        print("S", end = "")

if n != 0:
    for i in range(1, n + 1):
        print(i, end = "")